src/main.tsx

```diff
@@ -2,10 +2,24 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+// import { ReactQueryDevtools } from '@tanstack/react-query-devtools'
```

Comment:
leftover?

Reply:
I can remove it, but it was initially left to easily enable devtools.

Reply:
Leaving this so other can easily enable devtools.